Package.swift

```diff
@@ -113,6 +116,12 @@ let package = Package(
             dependencies: ["BraintreeCore"],
             publicHeadersPath: "Public"
         ),
+        .target(
+            name: "KountDataCollectorWrapper",
```

Comment:
For SPM we cannot call the target `KountDataCollector` as the [binary target and xcframework name need to match](https://stackoverflow.com/questions/71539013/spm-artifact-not-found-for-target-aaa-xcode-13-3-only). This does mean for SPM only, a merchant would need to do `import KountDataCollectorWrapper` in their `ViewController` while the other package managers could do `import KountDataCollector`. Should all be updated to call the wrapper module `KountDataCollectorWrapper`? Is there a different/better name for this module that wraps the Kount SDK?

Reply:
Sick, will update to `BraintreeKountDataCollector`